db/migrate/20200518135205_add_priority_to_todos.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriorityToTodos < ActiveRecord::Migration[6.0]
+  def change
+    add_column :todos, :priority, :integer
```

Comment:
since you're using an integer column, you can also make use of enums to humanize the interger value.
Read about enums in Rails